vhdbuilder/packer/install-dependencies.sh

```diff
@@ -540,6 +540,7 @@ mcr.microsoft.com/oss/kubernetes/ingress/nginx-ingress-controller:0.19.0
 mcr.microsoft.com/oss/virtual-kubelet/virtual-kubelet:1.2.1.1
 mcr.microsoft.com/azure-policy/policy-kubernetes-addon-prod:prod_20200901.1
```

Comment:
can we remove this?

Reply:
prod_20200901.1 image is still in use. prod_20200923.1 image is being deployed in AKS 20201012 bits.